charts/main-alb/templates/argocd-nginx.yaml

```
@@ -9,9 +9,23 @@ data:
       worker_connections 1024;
     }
     http {
+      # Rate limits per IP
+      limit_req_zone $binary_remote_addr zone=general:10m rate=10r/s;
+      limit_req_zone $binary_remote_addr zone=webhook:10m rate=5r/s;
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)